setup.py

```diff
@@ -12,13 +12,15 @@
     install_requires = [
         "django<1.7",
         "jsonobject-couchdbkit",
-        "dimagi-utils",
+        "dimagi-utils>=1.3usec",
         'django-soil',
         "openpyxl",
         "unidecode",
         "xlwt",
     ],
-    packages = find_packages(exclude=['*.pyc']),
-    include_package_data=True
+    packages=find_packages(exclude=['*.pyc']),
+    include_package_data=True,
+    dependency_links=[
+        'git+git://github.com/dimagi/dimagi-utils.git@2533a1aa96fd91a855b153e7bb9c15275ef844e7#egg=dimagi-utils-1.3usec',
```

Comment:
fun - we should probably bump the version once https://github.com/dimagi/dimagi-utils/pull/266 is merged